Code/GrpAttributes.m

```diff
@@ -187,6 +188,46 @@ elif Get(G,"solvable") and Get(G,"Agroup") then
     end if;
     return false;
 end intrinsic;
+*/
+
+intrinsic monomial(G::LMFDBGrp) -> BoolElt
+    {Determine if a group is monomial}
+    g:=G`MagmaGrp;
+    if not Get(G,"solvable") then
+        return false;
+    elif Get(G, "supersolvable") then
+        return true;
+    elif Get(G,"solvable") and Get(G,"Agroup") then
+        return true;
+    else
+        ct:=CharacterTable(g);
+        maxd := Integers() ! Degree(ct[#ct]); // Crazy that coercion is needed
+        stat:=[false : c in ct];
+        ls:= LowIndexSubgroups(G, maxd);
+        if Type(ls) eq NoneType then
+          return None();
```

Comment:
Shouldn't this be false?

Reply:
If we haven't computed enough low index subgroups, I think we should return None indicating that we don't know whether it's monomial or not.

Reply:
Yes, LowIndexSubgroups can return none. That is why I made the change. I hope the if the statement is correctly written, it did not give me an error when I run the SmallGroup command.